metals/src/main/scala/scala/meta/internal/worksheets/MdocEnrichments.scala

```diff
@@ -57,8 +57,21 @@ object MdocEnrichments {
    * continuation
    */
   def truncatify(statement: EvaluatedWorksheetStatement): String = {
-    if (statement.isSummaryComplete()) statement.summary()
-    else statement.summary() + "…"
+    if (statement.isSummaryComplete()) statement.prettySummary()
+    else statement.prettySummary() + "…"
+  }
+
+  /**
+   * Strips ANSI colors
+   */
+  def filerANSI(details: String): String =
+    details.replaceAll("\u001b\\[(\\d+;)*\\d{1,3}m", "")
```

Comment:
I've actually done this in the past in another place as well. We may want to make this generic and put it somewhere and then use it in both places.

https://github.com/scalameta/metals/blob/13be623fe994d152dd1ab264a9db400aefae85a3/metals/src/main/scala/scala/meta/internal/metals/debug/DebugProxy.scala#L216

Reply:
Oh, great. Thanks.